docs/get-started/v2/get-started/insights/network-performance.mdx

```diff
@@ -0,0 +1,117 @@
+
+
+
+### Frame Rate (FPS)
+The frame rate refers to the number of video frames transmitted per second by the local peer to the remote peers in a video call. 
+
+#### Frame rates and video quality
+- Higher outgoing frame rates result in smoother, less choppy video quality at the receiver's end. However, high frame rates require higher bitrates.
+- Lower outgoing frame rates allow the local peer to reduce their transmitted bitrate while maintaining visual quality.
+
+
+
+### Packet Loss
+Packet loss occurs when media packets being sent by the local peer fail to reach the remote peers, resulting in degraded audio/video quality of the peer. It occur due to network congestion, insufficient bandwidth, faulty network hardware, etc. 
+
+#### Measuring Packet loss
+- Some amount of data loss is inevitable, so the severity is measured by the percentage of total packets lost.
+- Packet loss below 5% is generally considered acceptable and normal.
+- Beyond 5%, high outgoing packet loss can significantly degrade the video resolution, frame rate, and image quality observed by remote viewers. Audio can be jittery or broken up or may miss sections of audio.
+
+
+### **Round Trip Time (RTT)**
+
+Measures the time in milliseconds (ms) it takes for a packet of data to travel from the source(client) to the destination and back again. 
+Higher the RTT, higher the latency.
+
+
+### Jitter
+
+Jitter refers to the variation in delay between delivered media packets being sent from the local peer to the remote peers. High outgoing jitter can cause received video to stutter or display jerky movements and make audio sound choppy or distorted to remote viewers.
+
+
+### Quality Limitations
+There are two main limitations that can severely impact the outgoing media quality transmitted from a local peer to remote peers:
+
+#### CPU Limitations
+
+Insufficient CPU cores or processing power can cause delays in encoding and transmitting outgoing audio/video packets. This leads to increased latency, jitter, and potential packet loss for the streams received by remote peers.
+
+#### Network Limitations
+
+Bandwidth, throughput, or network equipment constraints can create congestion and bottlenecks specifically along the local peer's upload path. This results in higher outgoing latency, jitter, and packet loss that significantly reduces the media quality observed by remote participants.
+
+</Tab>
+
+
+<Tab id='test-code-1'>
+
+### Bitrate
+When incoming media quality, bitrate is defined as the sum of all the bitrates published by all the remote peers in the call. It is defined using:
+
+- **Expected Bitrate**: The bitrate a client expects to receive based on the video resolution and layers selected according to the call template settings. This is configured for the role under templates.<br/>
```

Comment:
`...layers selected according to the call template setting` and `the number of tracks being downloaded`